### PR TITLE
ruget: add page

### PR DESCRIPTION
**Pull request by suyashbhawsar**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/ruget.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# ruget
+
+> Alternative to wget written in Rust.
+> More information: <https://github.com/ksk001100/ruget>.
+
+- Download the contents of a URL to a file (named "foo" in this case):
+
+`ruget https://example.com/foo`
+
+- Download the contents of a URL to a file with a specified [o]utput file (named "bar" in this case):
```
Comment (kbdharun) [inline pages/linux/ruget.md:10]:
```suggestion
- Download the contents of a URL to a specified [o]utput file:
```

File: pages/linux/ruget.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+`ruget --output bar https://example.com/foo`
```
Comment (kbdharun) [inline pages/linux/ruget.md:12]:
```suggestion
`ruget --output bar {{https://example.com/file}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution @suyashbhawsar, I have some suggestions to this page.

File: pages/linux/ruget.md (hunk 1 above)
Comment (suyashbhawsar) [reply pages/linux/ruget.md:10]: I think the 'specified' is a more precise word in this case. The other suggestions are good!

**Review by kbdharun — APPROVED** [review]

**Comment by suyashbhawsar** [comment]
@kbdharun I think 'specified' is more precise in this case than 'specific'.

**Comment by kbdharun** [comment]
> @kbdharun I think 'specified' is more precise in this case than 'specific'.

Yeah, That works too.

**Review by kbdharun — APPROVED** [review]

File: pages/linux/ruget.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+> Alternative to wget written in Rust.
+> More information: <https://github.com/ksk001100/ruget>.
+
+- Download the contents of a URL to a file:
+
+`ruget {{https://example.com/file}}`
+
+- Download the contents of a URL to a specified [o]utput file:
+
+`ruget --output filename {{https://example.com/file}}`
```
Comment (kbdharun) [inline pages/linux/ruget.md:12]: Just a small suggestion.

```suggestion
`ruget --output {{file_name}} {{https://example.com/file}}`
```

**Review by kbdharun — APPROVED** [review]

**Review by einverne — APPROVED** [review]
Looks good. Thank you for the new page.

**Pull request merged**